### DISCUSSION TITLE
Elite Dangerous

### DISCUSSION THREAD

**Discussion by RussianLioN** [Whisky-App/Whisky#212] [General]
My Favorite game is Elite Dangerous.
I launched it with CrossOver. 
But with Whisky it just doesn’t start. What are the ideas?

That’s what I found out about:

https://www.reddit.com/r/macgaming/comments/14i71yq/trying_to_play_elite_dangerous_via_the_game/

**Comment by coreyklass** [comment]
The Elite Dangerous launcher shows up as a blank window and you can't interact with it at all.

You can do one of two things:
1. Add the launcher parameters `/autorun /autoquit` and this will start Elite Dangerous Odyssey, bypassing the launcher problem.
2. Replace the bundled ED Launcher with the [Mini Elite Dangerous Launcher](https://github.com/rfvgyhn/min-ed-launcher) instead. This is what I'm currently doing, and Elite Dangerous runs perfectly.
